Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -109,7 +108,8 @@ class SettingSection : Setting {
 // A helper class for prefs that deal with sync. Handles reloading the tableView data if changes to
 // the fxAccount happen.
 private class AccountSetting: Setting, FxAContentViewControllerDelegate {
-    let settings: SettingsTableViewController
+    weak var settings: SettingsTableViewController!
```

Comment:
Note: Forgot to mention this in my comments but we should not be holding a strong reference here since the SettingsVC 'owns' these setting objects.